data/json/monsters/cyborgs.json

```diff
@@ -119,6 +119,54 @@
     "death_function": { "corpse_type": "BROKEN" },
     "flags": [ "SEES", "HEARS", "GOODHEARING", "NOHEAD", "NO_BREATHE", "PATH_AVOID_DANGER_2", "PRIORITIZE_TARGETS", "HIT_AND_RUN" ]
   },
+  {
+    "id": "mon_exodii_horse",
+    "type": "MONSTER",
+    "name": { "str": "Exodii horse" },
+    "description": "This is a bionically enhanced horse with shining metal armor protecting vital areas.",
+    "default_faction": "exodii",
+    "volume": "600 L",
+    "weight": "600 kg",
+    "species": [ "CYBORG" ],
+    "material": [ "steel" ],
+    "speed": 250,
+    "morale": 90,
+    "aggression": -20,
+    "hp": 125,
+    "symbol": "H",
+    "color": "light_gray",
+    "melee_skill": 8,
+    "melee_dice": 2,
+    "melee_dice_sides": 12,
+    "melee_cut": 0,
+    "dodge": 2,
+    "armor_bash": 12,
+    "armor_cut": 28,
+    "armor_acid": 12,
+    "armor_fire": 8,
+    "armor_bullet": 12,
+    "vision_day": 50,
+    "vision_night": 10,
+    "anger_triggers": [ "FRIEND_ATTACKED", "FRIEND_DIED", "HURT" ],
+    "placate_triggers": [ "PLAYER_WEAK" ],
+    "harvest": "mon_broken_cyborg",
```

Comment:
I am not sure horses should yield bionics for humans.

Reply:
Definitely no human bones anyway. I'll get a new harvest list together

Reply:
I actually tracked down this item group (it's hiding here https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/itemgroups/Monsters_Animals_Lairs/monster_drops_advtech.json ) and it's a mix of useful and terrible bionics, which I think gets at the concern about compatibility nicely.